HousingSearchListener/V1/HealthCheck/ElasticSearchHealthCheck.cs

```diff
@@ -0,0 +1,38 @@
+using System;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+using Nest;
+
+namespace HousingSearchListener.V1.HealthCheck
+{
+    public class ElasticSearchHealthCheck : IHealthCheck
```

Comment:
Whilst I love that you added this, it can't actually be access as yet, correct?

Reply:
Nope - good point. Not sure it can be added at any point actually.

Reply:
Up to you if you want to remove it for now.